framework/interface/api_handlers.py

```diff
@@ -117,6 +117,15 @@ def get(self):
         except exceptions.InvalidParameterType as e:
             raise tornado.web.HTTPError(400)
 
+class DashboardPanelHandler(custom_handlers.APIRequestHandler):
+    SUPPORTED_METHODS = ['GET']
+
+    def get(self):
+        try:
+            self.write(self.get_component("plugin_output").GetSeverityFrequency())
+        except exceptions.InvalidParameterType as e:
```

Comment:
If you are not using the Exception variable `e`, then simply do

`except exceptions.InvalidParameterType`